### DISCUSSION TITLE
Dual Finance - Saber Staking Options Proposal

### DISCUSSION THREAD

**Discussion by jlkramer16** [saber-hq/governance#5] [General]
Hi Saber Community, Dual Finance is designing the next generation of sustainable yield products to be the Base-Layer Incentivization Protocol for Web3. Our breakthrough token incentive mechanism, called Staking Options (SOs) aligns incentives of participants by granting those who provide liquidity or participate in governance, the right, but not the obligation to purchase tokens for a future price. No more token rewards, instead token option rewards. 

Dual Finance proposes the Saber community re-allocate its remaining Mining Reserve to be issued in SO form using our Staking Options Studio (SOS). The goal of this overhaul of incentives is to revitalize the Saber token economy and disrupt the negative cycle of inflationary incentives. The distribution of the remaining Mining Reserve will be split 80% into SOs for incentivizing LP token holders and 20% for locking up SBR into veSBR. 

The proposed implementation of LP SOs is 30 day expirations of 20% OTM (out-of-the-money) strikes, rolling every 3 days. Users will accrue SOs in proportion to the time and quantity they are staked. Gauge Weight Voting will allocate SOs in 2.5x the proportion as with spot token emissions. Effectively for every 1 SBR token previously earned, Users will earn 2.5x a 30 day call option to buy SBR at a 20% increase from the current spot price.

Participating in Gauge Weight Voting by locking up SBR will now also pay Users in SOs. Upon locking up 1 SBR for veSBR, users will receive 0.25 SOs expiring the same time they lockup until with a 100% OTM Strike and 25% Lockup Ratio. Current veSBR holders will receive a one time airdrop of SOs matching their current lock up period at 100% OTM strike and 25% Lockup Ratio as well.

Users can view and exercise these Staking Options at anytime by going to https://beta.dual.finance/balance and paying the exercise amount (strike price * quantity)  in USDC. All exercise proceeds minus a 3.5% fee go to the Saber Treasury for the community to grow the Saber ecosystem. For being an early partner and user of Staking Options, Dual Finance will issue a grant to the Saber treasury of 5M DUAL 1 year Staking Options with a strike of the greater of $0.25 & 100% OTM. Please let us know what questions or concerns you have, we are looking forward to a long and fruitful partnership with Saber!

To learn more about Dual Finance’s mission to re-incentivize crypto and bootstrap token options markets, follow us on Twitter (https://twitter.com/DualFinance), join the Discord (discord.gg/P3uH9AvEp5), and review our medium articles (https://medium.com/@dualfinance). Our Dual Investment Pools (DIPs) to earn sustainable, customized yield are also live for testing at https://beta.dual.finance/

**Comment by tmnxeq** [comment]
Thanks for taking the initiative. In principal I support the goal of the proposal

Random comments & questions:
* To my knowledge, the saber treasury is not controlled by the DAO, and we have no visibility on the team's dealings (e.g. wrt. to the Ecosystem Reserve spending). I don't trust that the team would not misappropriate funds. Full transparency & DAO control over the saber treasury needs to come first.
* What happens to the 3.5% exercise fee?
* What about sunSBR holder?

**Reply by jlkramer16** [reply]
Thanks for the feedback.

1. Full transparency & DAO control over the saber treasury is being assumed the case here, but would need core team to confirm the current setup.
2. The exercise fee is collected by the Dual Treasury which will be fully transparent and community owned from day 1. We hope to allocate exercise fees largely to talent capable of sourcing more projects who benefit from reinventing their reward mechanisms. 
3. Good catch, we'll aim to support sunSBR too on the governance implementation

**Comment by d0bby5** [comment]
I think SOs are a really interesting idea and there is a chance they could actually work as intended here. As tmnxeq pointed out, I have no idea what the treasury or remaining mining reserve look like, so it's difficult to gauge how large this proposed partnership would be. 

Just curious - how are the strike prices determined? Can you elaborate please?

Also, are you considering any partnerships with other DeFi protocols? Would love to see more projects offer incentives for governance participation.

**Reply by jlkramer16** [reply]
Appreciate the feedback. I will gather the specifics on the mining reserve amount from the core team.

Strike prices for LP SOs target more ATM (at-the-money) strikes and shorter expirations to incentivize more frequently exercise and smaller but more consistent returns compare to the governance SOs. The deltas of the options educates the strike selection and the resulting leverage multiple. The LP SOs are ~40 delta options meaning 40% of the time you expect them to be exercised hence why 2.5x is the leverage multiplier. For every prior 1 SBR in delta terms equals 2.5x SOs. If desirable, we can tweak the leverage down and offer closer to ATM options.

Yes we're targeting 2 other major Solana ecosystem projects as an initial batch of partnerships. Follow our twitter to be sure you don't miss those announcements. https://twitter.com/DualFinance

**Reply by d0bby5** [reply]
Appreciate the explanation here DonDuala! I've seen Dual Finance pop up in a few other discussions now. Look forward to learning more about it. Best of luck with securing partnerships!
